src/Game/Item.java

```diff
@@ -132,6 +133,42 @@ public static void queryDatabaseAndPatchItem(Connection c, String query) {
 		}
 	}
 	
+	/**
+	 * Patches discontinued edible item commands specified by {@link Settings#COMMAND_PATCH_TYPE}.
+	 * Removes completely the option to eat/drink
+	 */
+	public static void patchItemCommands() {
+		int commandPatchType = Settings.COMMAND_PATCH_TYPE;
+		int[] edible_rare_item_ids = { 246, 387, 422, 677 };
```

Comment:
We should leave in a comment what these item IDs are.

Reply:
fixed